source/index.html.md.erb

```diff
@@ -184,6 +184,34 @@ Parameter | Description
 --------- | -----------
 recruitment_cycle | 4-character year (e.g. 2019 for 2019/20 courses)
 
+### Get all changed courses
+
+This endpoint retrieves all courses changed since midnight UTC of the specified day.
+
+The returned results:
+
+- match the structure of the [Get all courses](#get-all-courses) endpoint
+- are sorted chronologically with the oldest update first
+- are paginated with a page size of 1000 (see the [pagination section](#pagination) for info about navigating pages)
+
+A course is marked as changed (and hence included in this endpoint) if:
+
+- the course itself has been changed
+- the campus status has changed
+- campus associations have changed
+- subject associations have changed
+
+#### HTTP Request
+
+`GET <%= config[:base_api_url] %>/<recruitment_cycle>/courses?changed_since=<YYYYMMDD>`
+
+#### URL Parameters
+
+Parameter | Description
+--------- | -----------
+recruitment_cycle | 4-character year (e.g. 2019 for 2019/20 courses)
+changed_since | Day in YYYYMMDD format
```

Comment:
Is there a reason we want to provide an interface to get changes with days-granularity? Do we not need/want to support higher-granularity updates? Every hour or N hours, for example?

I like ZenDesk's implementation for chat messages, and we're going to have to do something like that for paging anyway, so why not just do epoch seconds.

Reply:
My 2 cents is the thinking is:

- We're not going to have lots of updates happening, a changeset for a given slow day could be only a few dozen records potentially
- Splitting by days makes it easier to work with for mental arithmetic purposes (thinking in seconds isn't super easy)
- You can cache per day at the database level (though we probably won't need to)

Jake wrote down some more context from a chat we had today though so he might be able to chime in with better reasons.